demo/src/demos/PageLayout/PageLayoutStickyBarDemo.tsx

```diff
@@ -0,0 +1,96 @@
+/*---------------------------------------------------------------------------------------------
+ * Copyright (c) Bentley Systems, Incorporated. All rights reserved.
+ * See LICENSE.md in the project root for license terms and full copyright notice.
+ *--------------------------------------------------------------------------------------------*/
+import React from 'react';
+import { DemoHeader } from '../../common/DemoHeader';
+import { DemoSideNav } from '../../common/DemoSideNav';
+import { DemoTemplate } from '../../common/DemoTemplate';
+import { PageLayout } from '@itwin/itwinui-layouts-react';
+import { Text, Button, Surface, LabeledInput } from '@itwin/itwinui-react';
+
+const codeDemo = `import '@itwin/itwinui-layouts-css/styles.css';
+import { PageLayout } from '@itwin/itwinui-layouts-react';
+  
+const Demo = () => {
+  return (
+    <PageLayout>
+      <PageLayout.Header>
+        <DemoHeader />
+      </PageLayout.Header>
+
+      <PageLayout.SideNavigation>
+        <DemoSideNav />
+      </PageLayout.SideNavigation>
+
+      <PageLayout.Content padded>
+        <PageLayout.TitleArea>
+          <DemoTitleArea />
+        </PageLayout.TitleArea>
+        <PageLayout.ToolsArea
+          left={<DemoButton />}
+          right={<DemoButton />}
+        />
+        <Surface elevation={1} >
+          <DemoContent />
+        </Surface>
+      </PageLayout.Content>
+    </PageLayout>
+  );
+}`;
+
+export const PageLayoutStickyBarDemo = () => {
+  return (
+    <DemoTemplate title='App' codeExample={codeDemo}>
+      <PageLayout>
+        <PageLayout.Header>
+          <DemoHeader />
+        </PageLayout.Header>
+
+        <PageLayout.SideNavigation>
+          <DemoSideNav activeItemKey='Model' />
+        </PageLayout.SideNavigation>
+
+        <PageLayout.Content padded>
+          <PageLayout.TitleArea>
+            <Text variant='headline' as='h1'>
+              This is PageLayout with footer
+            </Text>
+          </PageLayout.TitleArea>
+          <form
+            style={{
+              display: 'flex',
+              flexDirection: 'column',
+              gap: 'var(--iui-size-s)',
+            }}
+          >
+            <LabeledInput label='Name' />
+            <LabeledInput label='Description' />
+            <LabeledInput label='Country' />
+            <LabeledInput label='City' />
+          </form>
+        </PageLayout.Content>
+        <PageLayout.ActionBar>
+          <Surface
```

Comment:
Can we put the visual styling on `.iui-layouts-page-content-action-bar` & remove `<Surface>`?  I think [this new `<Flex>` component](https://github.com/iTwin/iTwinUI/pull/1011) will help us achieve the centering & gap.

Reply:
To expand upon this, we should probably copy the horizontal padding rules that are applied to the body content area.  If the side nav is removed, the body content includes padding to accommodate for the notch & rounded edges of the screen.
![Screenshot 2023-01-20 at 12 03 07 PM](https://user-images.githubusercontent.com/849817/213760739-b6118c01-d9ee-4fbb-9d1f-5ab61bcebf3d.png)
![Screenshot 2023-01-20 at 12 05 57 PM](https://user-images.githubusercontent.com/849817/213760741-6f79a2d8-177c-49dd-a798-291996f55b4c.png)

Reply:
I added the notch padding to the action bar in [ca33911](https://github.com/iTwin/iTwinUI-layouts/pull/70/commits/ca339113fb9004b2bf76b47f41cbb085e8a27d5f).